### ISSUE TITLE
get_tasks without filter returns 400 Bad Request

### ISSUE THREAD

**Issue by isorakanon** [abhiz123/todoist-mcp-server#1]
The README shows an example of "Show all my tasks", but trying to get tasks without a filter returns a 400 Bad Request error.

Steps to reproduce:
1. Call todoist_get_tasks without any parameters
2. Also tried with just a limit parameter

Expected behavior:
- Should return all tasks when no filter is specified, as suggested by the README example "Show all my tasks"

Actual behavior:
- Returns "Error: Request failed with status code 400"

Note that the same function works correctly when a filter is specified (e.g., "today" or "next 7 days").

**Comment by nikhilmaddirala** [comment]
I'm getting this error all the time too

**Comment by jbohnslav** [comment]
Would appreciate merging the above PR

**Comment by justinotero** [comment]
Yup I keep getting `Error: Request failed with status code 400` with any combination of `getTask`

**Comment by abhiz123** [comment]
Already merged the fix - https://github.com/abhiz123/todoist-mcp-server/pull/5

**Comment by cenkalti** [comment]
Please update https://www.npmjs.com/package/@abhiz123/todoist-mcp-server. It is better to set up a GitHub Action to automate this process.
